app/src/main/kotlin/com/wire/android/ui/home/conversationslist/MentionScreen.kt

```diff
@@ -23,6 +23,7 @@ fun MentionScreen(
     onEditConversationItem: (ConversationItem) -> Unit,
     onScrollPositionChanged: (Int) -> Unit = {},
     onOpenUserProfile: (UserId) -> Unit,
+    openConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Comment:
maybe to follow the same pattern, we can rename it to 
```suggestion
    onOpenConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Reply:
done